### ISSUE TITLE
The app throws in IE11 because of includes(), find() and assign()

### ISSUE THREAD

**Issue by kachkaev** [isaachinman/next-i18next#107]
I received a report saying that that one of my websites no longer works in IE11. The investigation revealed that the reason for this is that I forgot a polyfill. Adding the following to the very beginning of `_app.tsx` resolves the issue:

```ts
import "core-js/fn/array/find";
import "core-js/fn/array/includes";
import "core-js/fn/object/assign";
```

IE11 is not so prevalent these days, so I'm not sure that polyfills for it need to be included into `next-i18next` by default. However, it might be still worth having a section in README that would mention browser support. In my case `core-js/fn/array/find` and `core-js/fn/array/find` were needed to run a Next.js app in general, while `core-js/fn/array/includes` became necessary only after I switched to `next-i18next` from a bespoke i18next integration. Usage of all three methods can be found in `next-i18next` source code.

If neither polyfilling nor adding a README section is desired, we can simply close this issue and keep it as an answer for those who struggle with IE and start searching for it in the repo. WDYT?

**Comment by isaachinman** [comment]
Not sure I follow. Are you not transpiling your deps into a vendor chunk? We might be able to change the [babel target](https://github.com/isaachinman/next-i18next/blob/master/.babelrc#L3) but in general users will be responsible for their own targets and browser support.

**Comment by kachkaev** [comment]
I'm not doing anything special in transpiling my Next.js app and actually don't believe that anything needs to be done in the source of `next-i18next` code as such. However, because calls to `Array.includes()`, `Array.find()` and `Object.assign()` inside the lib do cause issues in IE11, some mentioning of this situation might be worth it in README.

Actually one thing I completely forgot to mention is polyfilling of I18n API, which is used by `i18next`. My IE11-compatible Next.js + `next-i18next` app also includes this line in `_document.tsx`:

```tsx
<link rel="preload" href="https://cdn.polyfill.io/v2/polyfill.min.js?features=Map,fetch,Intl.~locale.ru,Intl.~locale.en&amp;unknown=polyfill" as="script"/>
```

Source: https://polyfill.io/

**Comment by isaachinman** [comment]
> We might be able to change the babel target

That's my proposed solution if it helps you. This project is transpiled via Babel prior to publish. We are not currently passing any targets into [`babel-preset-env`](https://babeljs.io/docs/en/babel-preset-env#targets).

> Sidenote, if no targets are specified, @babel/preset-env behaves exactly the same as @babel/preset-es2015, @babel/preset-es2016 and @babel/preset-es2017 together (or the deprecated babel-preset-latest).

**Comment by isaachinman** [comment]
We'd need [`useBuiltins`](https://github.com/babel/babel-preset-env#usebuiltins).

**Comment by isaachinman** [comment]
@kachkaev Should be fixed with 9c96479. Do you mind checking over my work and testing in IE11?

**Comment by kachkaev** [comment]
Thanks! I’ll give this a try in several days when there is a slot for that project.
